libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Queue/CSFNetwork.m

```diff
@@ -93,7 +93,7 @@ + (instancetype)currentNetwork {
 
 + (instancetype)networkForUserAccount:(SFUserAccount*)account {
     CSFNetwork *instance = nil;
-    if (!account.isTemporaryUser) {
+    if (account!=nil) {
```

Comment:
This is an area that's interesting to me.  Because while you can't set a `CSFNetwork` for the temporary user account, you currently _can_ set a network for "anonymous" and "guest" users, allowing network requests to endpoints that do not require authentication.

This change is going to require more consideration.